src/features/common/TreeCounter/TreeCounter.tsx

```diff
@@ -83,7 +83,8 @@ export default function TpoProfile(props: any) {
           <h1>{Sugar.Number.abbr(Number(props.planted), 1)}</h1>
           <h2>{t('me:treesPlanted')}</h2>
         </div>
-        {props.target && props.target !== 0 && (
+        
+        {props.target ? props.target !== 0 && (
```

Comment:
Where is the difference between `props.target !== 0` && and `props.target` - I wonder if the part `props.target !== 0 &&` might be useless?